### PR TITLE
new text added under the email tab

### PR DESCRIPTION
**Pull request by ramavavi**
Inserted text under the space provided to enter an email address : Update Text on Account Creation Page #469 

Issue Link: https://github.com/episphere/connectApp/issues/469

(https://user-images.githubusercontent.com/74061787/217064500-c5f409db-8a67-437d-ba83-902d662f180f.png)

### CODE REVIEW

File: js/pages/homePage.js [hunk 1]
```diff
@@ -450,6 +450,11 @@ export async function signInCheckRender ({ ui }) {
             signUpRender({ ui });
           });
 
+          const firebaseUiCardContentWrapper = document.querySelector('div[class~="firebaseui-relative-wrapper"]')
+          const pElement = document.createElement("p")
+          pElement.innerText = "Connect is a long-term study. Please use an email that you will be able to access in the future. Avoid using a work email if possible"
```
Comment (we-ai) [inline js/pages/homePage.js:455]:
```suggestion
          pElement.innerText = "Connect is a long-term study. Please use an email that you will be able to access in the future. Avoid using a work email if possible."
```

**Review by we-ai — COMMENTED** [review]
The font size is not consistent.You can adjust the style, take screenshots and discuss with people who requested this feature, and decide the final style.

File: js/pages/homePage.js (hunk 1 above)
Comment (we-ai) [reply js/pages/homePage.js:455]: Placing the text inside the card footer is better. Just personal opinion.

**Comment by ramavavi** [comment]
I confirmed with the people who asked for the change and told it looks fine, and will check how it works on mobile once its in Dev.

**Review by we-ai — APPROVED** [review]

**Pull request merged**